### PR TITLE
Add EL9 support

### PR DESCRIPTION
**Pull request by jcpunk**
#### Pull Request (PR) description
Add a set of defaults for EL9

#### This Pull Request (PR) fixes the following issues
N/A

### CODE REVIEW

**Comment by smortex** [comment]
I merged #33, can you please rebase on top of it?

**Comment by jcpunk** [comment]
Rebased

File: metadata.json [hunk 1]
```diff
@@ -37,7 +37,8 @@
       "operatingsystem": "RedHat",
       "operatingsystemrelease": [
         "7",
-        "8"
+        "8",
+        "9"
```
Comment (bastelfreak) [inline metadata.json:41]: I guess this also adds CentOS 9 support? can you add that as well to the metadata.json?

Comment (jcpunk) [reply metadata.json:41]: updated

**Review by bastelfreak — APPROVED** [review]

**Comment by jcpunk** [comment]
Can you publish a new forge version with these updates?

**Pull request merged**